src/index.ts

```diff
@@ -446,8 +455,8 @@ export function apply(ctx: Context, config: Config) {
           // event: newImage
           // id: 1
           // data:
-
-          if (res.headers.get('content-type') === 'application/x-zip-compressed') {
+          //                                                                        ↓ nai-v3
+          if (res.headers.get('content-type') === 'application/x-zip-compressed' || res.headers.get('content-disposition')?.includes('.zip')) {
```

Comment:
```suggestion
          if (res.headers.get('content-type') === 'application/x-zip-compressed' || res.headers.get('content-disposition')?.match(/filename=.+\.zip$/)) {
```

Would this be better than just checking the header string contains `.zip`?

这样会比只是检查头字串包含 `.zip` 更好吗？

Reply:
感觉差不多，你这么写还不如 endsWith 罢（

Reply:
Well nevermind.